src/modules/Admin/components/Rooms/AddRoom.tsx

```diff
@@ -0,0 +1,301 @@
+import { useState, useEffect } from "react";
+import {
+  Box,
+  TextField,
+  Button,
+  MenuItem,
+  Select,
+  Typography,
+  Stack,
+  FormHelperText,
+  List,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import { useForm } from "react-hook-form";
+import Dropzone from "react-dropzone";
+import axios from "axios";
+import { facility_Urls, roomsUrl } from "../../../../constants/End_Points";
+import styled from "@emotion/styled";
+import { useLocation, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+
+// Styled container for the form
+const FormContainer = styled(Box)`
```

Comment:
If this styles will be reused, we can move them into a seperate file and export them to other pages. This will also make this component much smaller.